samples/readme.md

```diff
@@ -0,0 +1,11 @@
+# .NET Interactive Samples
+
+This folder contains a number of samples that you can use to explore features of .NET Interactive.
+
+* [`docker-image`](docker-image/readme.md) contains a Dockerfile that generates an image with the latest .NET Interactive and Jupyter. This lets you try out .NET Interactive's Jupyter experience without needing to install Jupyter directly. 
```

Comment:
Should we say docker file instead? Just to set user expectations in the meantime.

Reply:
@colombod Any build-related fallout if we rename this folder to capture the purpose of this Docker setup vs the Binder one?

Reply:
yes this will break one of `Dockerfile_official_image` jon in the yaml file https://github.com/dotnet/interactive/blob/69f49f4e21a197100884456daa1442525fd73339/azure-pipelines.yml#L215

Reply:
the goal of such file is to 
`This lets you try out .NET Interactive's Jupyter experience without needing to install Jupyter directly. It can be used as base for packaging and redistribute docker images inside organization where end users are not allowed to install the components locally`

Reply:
I'm going to shave this yak another day, then, since it has a build impact. That justifies a separate PR I think.